src/parser/md.rs

```diff
@@ -557,12 +554,12 @@ impl MdParser {
     pub fn get_entry_points(
         &self,
         doc: &Document,
-        language: &Option<&str>,
+        language: Option<&str>,
     ) -> Vec<(String, String)> {
         let mut entries = vec![];
         let pref = &self.file_prefix;
-        for (name, _block) in doc.tree().code_blocks(language) {
-            if let Some(name) = name {
+        for block in doc.code_blocks(language) {
```

Comment:
@adamreichold A bug was introduced by this change, and by the change to the return type of function `code_blocks`. There was a reason to have the map... Currently investigating and trying to fix it.